backend/Dockerfile.celery

```diff
@@ -0,0 +1,25 @@
+FROM python:3.9-slim
+
+WORKDIR /app
+
+# Install system dependencies
+RUN apt-get update && apt-get install -y \
+  unzip \
+  gcc \
+  libffi-dev \
+  musl-dev \
+  && apt-get clean \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install Python dependencies
+COPY requirements.txt requirements.txt
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Install watchdog for auto-reloading
+RUN pip install watchdog[watchmedo]
```

Comment:
Is that just needed for local development? I guess for prod you wouldn't want watchdog right?

Reply:
I also believe it might make sense to use use uvicorn for this, it also has hot reloading built in: https://github.com/SlideSpeak/pdf-to-docx-server/blob/main/fastapi_server/Dev.Dockerfile#L16

the `Dev.Dockerfile` would be the one for developing and then there's a separate one for the prod stuff:
https://github.com/SlideSpeak/pdf-to-docx-server/blob/main/fastapi_server/Dockerfile

Reply:
Makes sense, I'll implement that 💯